src/metrics/metrics.md

```diff
@@ -2,10 +2,59 @@
 
 Here we provide definitions and information about our standard metrics.
 
-Table of contents with links to all metrics.
+## Table of Contents
+
+<!-- toc -->
 
 -----
 
+## Daily Active Users (DAU)
+
+### Overview
+
+DAU counts the number of unique profiles active in the product on each day.  This is intended to approximate the number of people using the product each day.
+
+### Details
+
+The concept of a "unique profile" may vary from product to product and the relationship between that concept and actual users may also vary.  As such care should be taken interpreting this metric as an actual count of users.
+
+The concept of being "active" is defined as in the [`usage criterion`](./usage.md) section and will vary by product.  A product may even have multiple possible `usage criteria`, but there will be one default choice that is our best approximation of general product usage.
+
+### Caveats
+
+If the number of users stays constant, but the average number of active profiles per user increases, this metric will tend to increase.
```

Comment:
I think we should go into more detail about this... somewhere.

There's this bit of good explanation on the distinction between pings and users here: https://docs.telemetry.mozilla.org/concepts/analysis_intro.html#caveats but it's somewhat incomplete and mixed in with discussion of an obsolete system (telemetry aggregates).

I can file a bug for a followup.

Reply:
Maybe this is a better thing to reference?  It probably needs to be updated, but isn't bad.  I'll put it in for now, but feel free to debate ;)

Reply:
Checking again, we have this which is actually reasonably up to date: https://docs.telemetry.mozilla.org/concepts/analysis_gotchas.html#profiles-vs-users

Are you ok with referencing that?

Reply:
yup; updated!